cfme/infrastructure/config_management/ansible_tower.py

```diff
@@ -0,0 +1,128 @@
+import attr
+from navmazing import NavigateToAttribute
+from widgetastic.widget import View
+
+from cfme.infrastructure.config_management import ConfigManagementCollectionView
+from cfme.infrastructure.config_management import ConfigManagementEntities
+from cfme.infrastructure.config_management import ConfigManagementSideBar
+from cfme.infrastructure.config_management import ConfigManagementToolbar
+from cfme.infrastructure.config_management import ConfigManagerProvider
+from cfme.infrastructure.config_management.config_profiles import ConfigProfilesCollection
+from cfme.infrastructure.config_management.config_systems import ConfigSystem
+from cfme.utils import conf
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigator
+from widgetastic_manageiq import Search
+
+
+class AnsibleTowerProvidersAllView(ConfigManagementCollectionView):
+    """The main list view"""
+    toolbar = View.nested(ConfigManagementToolbar)
+    sidebar = View.nested(ConfigManagementSideBar)
+    search = View.nested(Search)
+    including_entities = View.include(ConfigManagementEntities, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        title_text = 'All Ansible Tower Providers'
+        return (
+            self.in_config and
+            self.entities.title.text == title_text
+        )
+
+
+class ConfigSystemAllView(AnsibleTowerProvidersAllView):
+    """The config system view has a different title"""
+
+    @property
+    def is_displayed(self):
+        title_text = 'All Ansible Tower Configured Systems'
+        return (
+            self.in_config and
+            self.entities.title.text == title_text
+        )
+
+
+@attr.s(cmp=False)
```

Comment:
https://github.com/ManageIQ/integration_tests/pull/9691

Following @izapolsk example above, use `eq`, not `cmp` here, as `cmp` is deprecated in attrs.

Reply:
done